src/devices/sonoff.ts

```diff
@@ -622,6 +622,34 @@ const definitions: Definition[] = [
                 unit: 'mV',
                 access: 'STATE_GET',
             }),
+            numeric({
+                name: 'valve_opening_degree(version >= v1.1.4)',
+                cluster: 0xFC11,
+                attribute: {ID: 0x600B, type: 0x20},
+                description: 'Valve open position (percentage) control. ' +
+                    'If the opening degree is set to 100%, the valve is fully open when it is opened. ' +
+                    'If the opening degree is set to 0%, the valve is fully closed when it is opened, ' +
+                    'and the default value is 100%, ' +
+                    'The valve opening degree should be greater than or equal to the valve closing degree.',
+                valueMin: 0.0,
+                valueMax: 100.0,
+                valueStep: 1.0,
+                unit: '%',
+            }),
+            numeric({
+                name: 'valve_closing_degree(version >= v1.1.4)',
```

Comment:
Please move `(version >= v1.1.4)` to the `description` (same for `valve_opening_degree`)

Reply:
Thanks for your reminding, it has been modified

Reply:
The version of which component does this refer to exactly?